bundle/src/projects/commercial/modules/header-bidding/request-bids.ts

```diff
@@ -8,7 +8,10 @@ const retainTopAboveNavSlotSize = (
 	advertSize: Advert['size'],
 	hbSlot: HeaderBiddingSlot,
 ): HeaderBiddingSlot[] => {
-	if (hbSlot.key !== 'top-above-nav') {
+	if (
+		hbSlot.key !== 'top-above-nav' &&
+		!hbSlot.key.includes('front-banner')
```

Comment:
This is a very minor suggestion but `.startsWith` might read better here

Reply:
Good idea